deploy/monitor/push_prometheus_rule.yaml

```diff
@@ -14,15 +14,15 @@ spec:
   groups:
     - name: general.rules
       rules:
-      - alert: UnifiedpushDown
-        expr: absent(kube_pod_container_status_running{namespace="unifiedpush",container="ups"}==1)
+      - alert: UnifiedPushDown
+        expr: absent(kube_pod_container_status_running{namespace="unifiedpush",container="ups"} == 1)
```

Comment:
It's better to change it to `>= 1` in case the pods are scaled up.

Reply:
Sure  and well done.